build.gradle

```diff
@@ -90,18 +91,24 @@ distributions {
 
 dependencies {
     compileOnly "org.apache.kafka:connect-api:$kafkaVersion"
-    compileOnly "org.apache.kafka:connect-runtime:$kafkaVersion"
+    implementation "org.apache.kafka:connect-runtime:$kafkaVersion"
```

Comment:
I wonder what was the reason to change `connect-runtime` to `implementation` but leave `connect-api`?

Reply:
I'll explain why it matters: with `implementation`, `connect-runtime` gets into the library's dependencies and--transitively--into GCS and S3 connectors. It will be packed along with other dependency JARs in the distro, placed into Kafka Connect plugins directory. The question is: Do we need to have `connect-runtime` classes additionally provided? Do we also really want `1.1.0` Connect runtime to be mixed together with `2.x`?

Reply:
True forget to remove ... It was just for small test which I made